Makefile

```diff
@@ -221,6 +221,9 @@ run-kubernetes-master: stop-kubernetes-master
 		--server=http://localhost:8080 \
 		apply -f /manifests/test/namespaces.yaml
 
+	# Wait until some resources we expect to exist are present.
```

Comment:
Spotted this when trying to run some tests with a focus - it takes 10-15 secs for the SA to show up.

We get away with it if we run all the tests at once, because we get lucky with ordering. but if you run a test that expects the SA to exist first, then it fails!